### DISCUSSION TITLE
Prerequisites to joining secret scanning partner program

### DISCUSSION THREAD

**Discussion by duhuan** [community/community#50201] [Enterprise]
### Select Topic Area

Question

### Body

Hi, Github experts
I notice there's a [program for partners](https://docs.github.com/en/code-security/secret-scanning/secret-scanning-partner-program) to scan secrets. This will greatly improve our customers' security if we can join the program. But according to the [doc](https://docs.github.com/en/code-security/secret-scanning/about-secret-scanning), it seems we have to buy GitHub Enterprise Cloud first.
To be clear, should we buy the enterprise license, or we can directly start a thread with [secret-scanning@github.com](mailto:secret-scanning@github.com) to join the program?

**Comment by duhuan** [comment]
Up, anyone can help me?

**Comment by naimur-reza** [comment]
Based on the information available on the GitHub website, it appears that the Secret Scanning feature is only available to organizations on the GitHub Enterprise Cloud plan. This means that you would need to purchase the Enterprise Cloud plan in order to use the Secret Scanning feature.

However, if you are not sure about the requirements or if you have any questions about the Secret Scanning program, it may be a good idea to reach out to the GitHub support team directly. You can contact them by starting a thread with [secret-scanning@github.com](mailto:secret-scanning@github.com) or by using the contact form on the GitHub website. They will be able to provide you with more information and guidance on how to join the program.

**Reply by duhuan** [reply]
Thank you @naimur-reza , it does help me!  I'll contact githhub by mail.

**Comment by 15MariamS** [comment]
Hi @duhuan, 

Any service provider can join our secret scanning partner program, for free! If you create tokens for users, you'd be a good fit. You can reach out to secret-scanning@github.com to get next steps from our team.

**Reply by duhuan** [reply]
awesome! Great to know this!
